From 1767590e5da1817e0f9ea0521af9096acb2590c6 Mon Sep 17 00:00:00 2001
From: Steve Pavao <stevep@korgrd.com>
Date: Fri, 10 Aug 2018 17:09:50 -0400
Subject: [PATCH 366/454] devicetree: add RPi CM3 dts to arm64; mimic the RPi
 3B arm64 dts implementation, by referring to the actual dts file in the arm
 directory

---
 arch/arm64/boot/dts/broadcom/Makefile            | 2 ++
 arch/arm64/boot/dts/broadcom/bcm2710-rpi-cm3.dts | 3 +++
 2 files changed, 5 insertions(+)
 create mode 100644 arch/arm64/boot/dts/broadcom/bcm2710-rpi-cm3.dts

--- a/arch/arm64/boot/dts/broadcom/Makefile
+++ b/arch/arm64/boot/dts/broadcom/Makefile
@@ -9,6 +9,8 @@ dtb-$(CONFIG_ARCH_BCM2835) += bcm2837-rp
 dtb-$(CONFIG_ARCH_BCM2709) += bcm2710-rpi-3-b.dtb
 dtb-$(CONFIG_ARCH_BCM2835) += bcm2710-rpi-3-b.dtb
 dtb-$(CONFIG_ARCH_BCM2835) += bcm2710-rpi-3-b-plus.dtb
+dtb-$(CONFIG_ARCH_BCM2709) += bcm2710-rpi-cm3.dtb
+dtb-$(CONFIG_ARCH_BCM2835) += bcm2710-rpi-cm3.dtb
 
 dts-dirs += ../overlays
 
--- /dev/null
+++ b/arch/arm64/boot/dts/broadcom/bcm2710-rpi-cm3.dts
@@ -0,0 +1,3 @@
+#define RPI364
+
+#include "../../../../arm/boot/dts/bcm2710-rpi-cm3.dts"
